tsa/tsacmd/command.go

```diff
@@ -141,6 +144,46 @@ func (cmd *TSACommand) Runner(args []string) (ifrit.Runner, error) {
 		httpClient:        httpClient,
 		sessionTeam:       sessionAuthTeam,
 	}
+	// Starts a goroutine that his purpose is to basically listen to the
+	// SIGUSR1 syscall to then reload the config.
+	// For now it only reload the TSACommand.AuthorizedKeys but any
+	// other configuration could be added
+	go func() {
+		for {
+
+			// Set up channel on which to send signal notifications.
+			// We must use a buffered channel or risk missing the signal
+			// if we're not ready to receive when the signal is sent.
+			c := make(chan os.Signal, 1)
+			signal.Notify(c, syscall.SIGUSR1)
```

Comment:
From my experience, SIGHUP tends to be more commonly used for "hey daemon, reload your config" overall, and SIGUSR1 has a very specific purpose for workers (which would mean that if we want to extend the idea of "reload on signal received", this would collide with the `land`-on-SIGUSR1 logic we currently have there).

Wdyt of going w/ SIGHUP?

Reply:
I agree. I updated the PR and commits to use SIGHUP